installer/pkg/cmd/gcp_broker.go

```diff
@@ -43,7 +43,7 @@ func NewAddGCPBrokerCmd() *cobra.Command {
 		Long:  `Adds a GCP broker to Service Catalog`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := addGCPBroker(); err != nil {
-				fmt.Println("failed to configure GCP broker: %v", err)
+				fmt.Println("failed to configure GCP broker")
```

Comment:
is this better than:

return fmt.Errorf("... %v", err)

?

Reply:
I just followed the pattern all the other commands do. Examples are like below. IMO, the first one is (a little bit) more clear because it uses keyword "Error", and separates the error message and the result message.

If we do 

```
fmt.Println("XXX")
return err
```

The stdout/stderr will display:

```
XXX
Error: <err.error()>
```

If we do

```
return fmt.Errorf("XXX: %v", err)
```

The stdout/stderr will display:

```
XXX: <err.error()>
```